docs/concepts/governance/maintainers.md

```diff
@@ -0,0 +1,17 @@
+# Maintainers
+
+This document lists the Maintainers of the Project. Maintainers may be added once approved by the existing maintainers as described in the [Governance document](policy.md). By adding your name to this list you are agreeing to abide by the Project governance documents and to abide by all of the Organization's polices, including the [code of conduct](https://github.com/pyscript/governance/blob/main/CODE-OF-CONDUCT.md), [trademark policy](https://github.com/pyscript/governance/blob/main/TRADEMARKS.md), and [antitrust policy](https://github.com/pyscript/governance/blob/main/TRADEMARKS.md). If you are participating because of your affiliation with another organization (designated below), you represent that you have the authority to bind that organization to these policies.
```

Comment:
"represent" isn't really working grammatically here, though I understand the goal of it... perhaps:

"... you authorize the binding of that organization to these policies."

Reply:
Since @kevingoldsmith put all these governance docs together, I'd like his input on that change before we make it.

Reply:
represent comes from the original github document. It is meant to be legalese. I wouldn't change it if it is a bit janky but the meaning is clear.